tools/quantp/test_data/protein transcript correlation on protein_data_tabular and transcript_data_tabular.html

```diff
@@ -0,0 +1,16 @@
+<font color="red">*Note that <u>correlation</u> is <u>sensitive to outliers</u> in the data. So it is important to analyze outliers/influential observations in the data.<br> Below we use <u>Cook's distance based approach</u> to identify such influential observations.</font></td></table><hr/><h2 id="regression_data"><font color=#ff0000>REGRESSION ANALYSIS</font></h2><font><h3>Linear Regression model fit between Proteome and Transcriptome data</h3></font> <p>Assuming a linear relationship between Proteome and Transcriptome data, we here fit a linear regression model.</p> <table  border=1 cellspacing=0 cellpadding=5 style="table-layout:auto; "> <tr bgcolor="#7a0019"><th><font color=#ffcc33>Parameter</font></th><th><font color=#ffcc33>Value</font></th></tr><tr><td>Formula</td><td> PE_abundance~TE_abundance </td></tr> <tr><td colspan='2' align='center'> <b>Coefficients</b></td> </tr> <tr><td> (Intercept) </td><td> -0.06910598  (Pvalue: 1.220723e-05 ) </td></tr> <tr><td> TE_abundance </td><td> 0.1712395  (Pvalue: 4.168015e-10 ) </td></tr> <tr><td colspan='2' align='center'> <b>Model parameters</b></td> </tr>
```

Comment:
@pravs3683 please do not include files with spaces in the names, this can break in strange ways. Use _ instead.

Reply:
I was wondering that space would create a problem. Changed the file name to "output.html" and it is also present in the "test_data" folder now.